yarn.lock

```diff
@@ -1033,13 +1033,20 @@
     core-js-pure "^3.20.2"
```

Comment:
Are these changes from the chassis lib?

Reply:
I am not sure about that. This is a file that is auto generated and the updates received by this files are due to the changes in the package.json where react-router-v6 upgrade changes were made. Do you notice any issues due to the changes?